src/main/scala/firrtl/stage/FirrtlCli.scala

```diff
@@ -0,0 +1,23 @@
+// See LICENSE for license details.
+
+package firrtl.stage
+
+import firrtl.options.Shell
+
+/** [[firrtl.options.Shell Shell]] mixin that provides command line options for FIRRTL. This does not include any
+  * [[firrtl.options.RegisteredLibrary RegisteredLibrary]] or [[firrtl.options.RegisteredTransform RegisteredTransform]]
+  * as those are automatically loaded by the [[firrtl.options.Stage Stage]] using this [[firrtl.options.Shell Shell]].
```

Comment:
Duplicated names here again, is this just a rendering thing?

Reply:
I think the names are okay here. Both `RegisteredLibrary` and `RegisteredTransform` should likely be mentioned.